.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_from:
+  - https://shopify.github.io/ruby-style-guide/rubocop.yml
```

Comment:
We should use the `rubocop-shopify` gem and inherit from the gem. See https://github.com/Shopify/ruby-style-guide#ruby-style-guide

Reply:
Done!